scriptworker/test/test_context.py

```diff
@@ -90,3 +92,26 @@ def test_temp_queue(context, mocker):
     assert taskcluster.async.Queue.called_once_with({
         'credentials': context.temp_credentials,
     }, session=context.session)
+
+
+@pytest.mark.asyncio
+async def test_projects(context, mocker):
```

Comment:
I'm not sure to follow what this test does in regard to expiration date.

Reply:
I noticed we were missing some coverage without integration tests, possibly after I made some artifact test changes.